### PR TITLE
adding compat data for css at-rules media any-hover

### PR DESCRIPTION
**Pull request by nathancharles**
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/any-hover

### CODE REVIEW

File: css/at-rules/media.json [hunk 1]
```diff
@@ -264,6 +264,61 @@
...
+              },
+              "opera": {
+                "version_added": "28"
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "9",
+                "notes": "Implemented in <a href='https://bugs.webkit.org/show_bug.cgi?id=134822' rel='noopener'>WebKit bug #134822</a>."
```
Comment (ddbeck) [inline css/at-rules/media.json]: Would you mind removing this note? Generally, notes that are about historical bugs (i.e., features that have shipped and don't have any known caveats) don't need to be migrated.

File: css/at-rules/media.json [hunk 2]
```diff
@@ -264,6 +264,61 @@
...
             }
           }
+        },
+        "any-hover": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/any-hover",
+            "description": "<code>any-hover</code> media feature",
+            "support": {
+              "webview_android": {
+                "version_added": false
```
Comment (ddbeck) [inline css/at-rules/media.json]: I did [a little digging](https://www.chromestatus.com/features/6460705494532096) and it looks like Android Webview, Chrome, and Chrome for Android all shipped this in version 41. Would you mind updating all three values?

File: css/at-rules/media.json [hunk 3]
```diff
@@ -264,6 +264,61 @@
...
+                "notes": "<a href='https://bugzil.la/1035774' rel='noopener'>bug 1035774 – Implement Interaction Media Features including pointer:coarse that replaces non-standard -moz-touch-enabled</a>"
+              },
+              "ie": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": "28"
+              },
+              "opera_android": {
+                "version_added": null
```
Comment (ddbeck) [inline css/at-rules/media.json]: This is also 28.

File: css/at-rules/media.json [hunk 4]
```diff
@@ -264,6 +264,61 @@
...
+              },
+              "edge": {
+                "version_added": "16"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "<a href='https://bugzil.la/1035774' rel='noopener'>bug 1035774 – Implement Interaction Media Features including pointer:coarse that replaces non-standard -moz-touch-enabled</a>"
```
Comment (ddbeck) [inline css/at-rules/media.json]: For this note and the one below, it's probably best to replace it with something like `"See <a href='https://bugzil.la/1035774'>bug 1035774</a>."` Bug titles can and do change, after all, so this is one less thing to go out of date.

File: css/at-rules/media.json [hunk 5]
```diff
@@ -264,6 +264,61 @@
...
+              },
+              "firefox_android": {
+                "version_added": false,
+                "notes": "<a href='https://bugzil.la/1035774' rel='noopener'>bug 1035774 – Implement Interaction Media Features including pointer:coarse that replaces non-standard -moz-touch-enabled</a>"
+              },
+              "ie": {
+                "version_added": null
```
Comment (ddbeck) [inline css/at-rules/media.json]: This is `false`.

File: css/at-rules/media.json [hunk 6]
```diff
@@ -264,6 +264,61 @@
...
+                "version_added": "41"
+              },
+              "chrome_android": {
+                "version_added": "50"
+              },
+              "edge": {
+                "version_added": "16"
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (ddbeck) [inline css/at-rules/media.json]: This is the same as edge, 16.

**Comment by Elchi3** [comment]
Hey @nathancharles, I've addressed Daniel's review comments for you. Thanks for your contribution!

@ddbeck does this look good now?

**Comment by ddbeck** [comment]
Yep, looks good! Thank you @Elchi3! I'll merge it now.

**Comment by nathancharles** [comment]
Thanks @Elchi3!

**Pull request merged**